app/models/hbx_enrollment.rb

```diff
@@ -38,6 +38,13 @@ class HbxEnrollment
   WAIVED_STATUSES     = %w(inactive renewing_waived)
 
   ENROLLED_AND_RENEWAL_STATUSES = ENROLLED_STATUSES + RENEWAL_STATUSES
+  TERM_REASONS = %w[non_payment voluntary_withdrawl retroactive_canceled].freeze
+
+  module TermReason
```

Comment:
is this relevant ? i don't see this being used anywhere else

Reply:
In DC It's been used for reinstate feature, I can remove now and we can add it back while implementing reinstate operation.